geth/jail/jail.go

```diff
@@ -67,6 +70,17 @@ func (jail *Jail) NewCell(chatID string) (common.JailCell, error) {
 	return cell, nil
 }
 
+// Stop stops jail and all assosiacted cells.
+func (jail *Jail) Stop() {
+	jail.cellsMx.Lock()
+	defer jail.cellsMx.Unlock()
+
+	for _, cell := range jail.cells {
+		cell.Stop()
+	}
+	jail.cells = nil
```

Comment:
@divan 
The next call to JailManager().NewCell will cause a panic
Let me change it to `jail.cells = make(map[string]*Cell)` into #382

Reply:
You're absolutely right: https://github.com/status-im/status-go/pull/398